src-docs/src/index.js

```diff
@@ -2,27 +2,29 @@ import React, { createElement } from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router, Switch, Route, Redirect } from 'react-router';
+import { Helmet } from 'react-helmet';
 
 import configureStore, { history } from './store/configure_store';
 
-import { AppContainer } from './views/app_container';
+import { AppContext } from './views/app_context';
+import { AppView } from './views/app_view';
 import { HomeView } from './views/home/home_view';
 import { NotFoundView } from './views/not_found/not_found_view';
 import { registerTheme, ExampleContext } from './services';
 
 import Routes from './routes';
-import themeLight from './theme_light.scss';
-import themeDark from './theme_dark.scss';
-import themeAmsterdamLight from './theme_amsterdam_light.scss';
-import themeAmsterdamDark from './theme_amsterdam_dark.scss';
+import legacyThemeLight from './legacy_light.scss';
+import legacyThemeDark from './legacy_dark.scss';
+import themeAmsterdamLight from './theme_light.scss';
+import themeAmsterdamDark from './theme_dark.scss';
```

Comment:
Any particular reason these still reference "Amsterdam"?

```suggestion
import themeLight from './theme_light.scss';
import themeDark from './theme_dark.scss';
```